### PR TITLE
Integrate lamassu-coins module

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Create the lamassu-coins package](https://trello.com/c/OwjinDcp/486-create-the-lamassu-coins-package)

**Review by jsousa127 — APPROVED** [review]

**Comment by chaotixkilla** [comment]
[ERC-20: Add USDT](https://trello.com/c/snrPYh1G/549-erc-20-add-usdt)

**Review by josepfo — APPROVED** [review]

File: lib/brain.js [hunk 1]
```diff
@@ -11,12 +11,13 @@ const pify = require('pify')
 const pDelay = require('delay')
 const pAny = require('p-any')
 const pSettle = require('p-settle')
+const supportedCoins = require('lamassu-coins').CRYPTO_CURRENCIES
+const coinUtils = require('lamassu-coins').utils
```
Comment (RafaelTaranto) [inline lib/brain.js:15]:
```suggestion
const { utils as coinUtils, CRYPTO_CURRENCIES } = require('lamassu-coins')
```

Comment (RafaelTaranto) [reply lib/brain.js:15]: Same thing for the rest of the places with the import.

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**